privaterelay/signals.py

```diff
@@ -55,8 +55,8 @@ def invitations_only(sender, **kwargs):
         return True
 
     except Invitations.DoesNotExist:
-        waitlist_invite = Invitations.objects.filter(
-            Q(email=email) | Q(fxa_uid=fxa_uid), active=False
+        waitlist_invite = get_invitation(
+            email=email, fxa_uid=fxa_uid, active=False
```

Comment:
Calling this in the `Inviations.DoesNotExist` block means that `get_invitation` also raises the same `DoesNotExist`.

Reply:
Nevermind. The above check is for `active=True`, this one is for `active=False` and is OUTSIDE the `try`.